### ISSUE TITLE
downloaden en vergridden scheiden

### ISSUE THREAD

**Issue by OnnoEbbens** [ArtesiaWater/nlmod#47]
Op dit moment worden de download en vergrid functies vaak in één keer aangeroepen. Bijv. in de functies `nlmod.read.ahn.get_ahn()` en `nlmod.read.knmi.get_recharge()`. Het zou handig zijn om deze functies van elkaar te kunnen scheiden. Dus dat je in het project eenmaal de data die je nodig hebt download en daarna naar deze data verwijst en deze vergrid bij het maken van een specifiek model.

Je kan dan bijv. het AHN van heel je projectgebied op fijne schaal downloaden (eenmalig en langzaam) en deze voor verschillende modellen vergridden (snel).

**Comment by dbrakenhoff** [comment]
Naming:
- all functions that download source data should start with `download_*`
- think about name for regridding data array to model coordinates and adding to dataset

**Comment by dbrakenhoff** [comment]
Move surface_water download methods to `nlmod/read/`

**Comment by OnnoEbbens** [comment]
Add this workflow to example notebooks.

Still create a way to this for knmi data

**Comment by OnnoEbbens** [comment]
This is fixed for KNMI data. Only task left is to create an example notebook where we first download all the data and then use it.

**Comment by OnnoEbbens** [comment]
Yes I agree with the renaming. Currently we have 3 types of functions:
1. Download data: `download_..`
2. Fit the data to the model dimensions, return a dataset
3. Do both 1 and 2. The current `get_..` functions

We can think about whether we want to keep supporting the 3th. I think we should at least for a bit for backward compatibility.

Also some functions are now a combination of 2 and 3. So it will require a bit of restructuring the read modules to be able to rename them properly.

**Comment by dbrakenhoff** [comment]
I think 3
